pkg/machines/actions.es6

```diff
@@ -83,9 +91,16 @@ export function vmDesktopConsole(vm, consoleDetail) {
  * and scheduled on later.
  *
  * @param action I.e. getAllVms()
+ * @param timeout Non-default timeout
+ * @param name VM name (if nonexisting or usage not enabled, action will not be run);
+ *             if not given, use global timeout
+ * @param connectionName Connection for #name
  */
-export function delayPolling(action, timeout) {
+export function delayPolling(action, timeout, name, connectionName) {
     return (dispatch, getState) => {
+        if (name && !usagePollingEnabled(getState(), name, connectionName))
```

Comment:
Also, if `delayPolling` calls itself directly further down ("since refreshing is switched off"), it does so without passing `name` and `connectionName`.  Not sure if that matters but it is in inconsistent, no?